llvm/test/CodeGen/PowerPC/fp128-libcalls.ll

```diff
@@ -44,6 +53,14 @@ define fp128 @extenddfkf2(double %a) {
   ret fp128 %1
 }
 
+define half @trunctfhf2(fp128 %a) {
+; CHECK-LABEL: trunctfhf2:
+; CHECK: __trunctfhf2
```

Comment:
I think this needs an update at https://github.com/llvm/llvm-project/blob/76393d3863bf7394edb69fb6a371b9a1c2c09ff6/llvm/lib/IR/RuntimeLibcalls.cpp#L85-L86 so `__trunckfhf2` is used rather than the tf version.